assets/src/utils/Mixin.js

```diff
@@ -86,6 +86,74 @@ export default {
             return match;
         },
 
+        weposConfirmAlert( { title = '', text = '' } ) {
```

Comment:
You don't need to prefix it with `wepos`, using it will make it more verbose. Just `confirmAlert` is sufficient.